sass/layout/_site.sass

```diff
@@ -36,47 +36,11 @@ body > div
     +pre($logo-width,$brand-width)
     text-transform: none
 
-#read-only-banner
-  background: rgba(0, 0, 0, 0.7)
-  border: 1px solid #000
-  +border-radius(6px, 6px)
-  +box-shadow(rgba(0, 0, 0, 0.2), 0 0 8px)
-  color: #fff
-  left: 32px
-  padding: 12px 20px 12px 30px
-  position: fixed
-  text-shadow: 1px 1px 1px #000
-  top: 32px
-  z-index: 9999
-
-  a,
-  a:visited
-    color: #ffcc00
-
-  .close-button
-    background: #000
-    border: 2px solid #fff
-    +border-radius(15px, 15px)
-    +box-shadow(rgba(0, 0, 0, 0.2), 0 0 8px)
-    cursor: pointer
-    height: 24px
-    left: -8px
-    position: absolute
-    text-indent: -9999px
-    top: -8px
-    width: 24px
+#read-only-message
+  border: 2px solid red
+  color: red
```

Comment:
My eyes! they're bleeding! :)

That red is too stark. Can you change both to `darkred`. If you do, it'll look like this:
![screenshot 2015-01-16 08 47 22](https://cloud.githubusercontent.com/assets/26739/5780053/55eb0174-9d5c-11e4-9f8d-50009f379230.png)